opensrp-chw-core/src/test/java/org/smartregister/chw/core/activity/CoreBirthNotificationRegisterActivityTest.java

```diff
@@ -0,0 +1,141 @@
+package org.smartregister.chw.core.activity;
+
+import android.app.ProgressDialog;
+
+import org.apache.commons.lang3.tuple.Triple;
+import org.json.JSONObject;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.powermock.reflect.Whitebox;
+import org.robolectric.util.ReflectionHelpers;
+import org.smartregister.Context;
+import org.smartregister.chw.core.contract.CoreChildRegisterContract;
+import org.smartregister.chw.core.presenter.CoreChildRegisterPresenter;
+
+import java.lang.ref.WeakReference;
+
+import timber.log.Timber;
+
+public class CoreBirthNotificationRegisterActivityTest {
+
+    @Mock
+    private CoreChildRegisterContract.Model model;
+    @Mock
+    private JSONObject jsonObject;
+    @Mock
+    private CoreChildRegisterContract.InteractorCallBack callBack;
+    @Mock
+    private CoreChildRegisterContract.Interactor interactor;
+    private CoreChildRegisterPresenter presenter;
+    @Mock
+    private CoreChildRegisterContract.View view;
+    private CoreBirthNotificationRegisterActivity coreBirthNotificationRegisterActivity;
+    @Mock
+    private ProgressDialog progressDialog;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+        coreBirthNotificationRegisterActivity = Mockito.mock(CoreBirthNotificationRegisterActivity.class, Mockito.CALLS_REAL_METHODS);
+        presenter = Mockito.mock(CoreChildRegisterPresenter.class, Mockito.CALLS_REAL_METHODS);
+        ReflectionHelpers.setField(presenter, "viewReference", new WeakReference<>(view));
+        ReflectionHelpers.setField(presenter, "interactor", interactor);
+        ReflectionHelpers.setField(presenter, "model", model);
+    }
+
+    @Test
+    public void testStartMeInitiatesActivity() {
+        CoreBirthNotificationRegisterActivity.startMe(coreBirthNotificationRegisterActivity);
+        Mockito.verify(coreBirthNotificationRegisterActivity).startActivity(Mockito.any());
+    }
+
+    @Test
+    public void testActivityLoaded() {
+        Assert.assertNotNull(coreBirthNotificationRegisterActivity);
+    }
+
+    @Test
+    public void testInitializePresenter() {
+        coreBirthNotificationRegisterActivity.initializePresenter();
+        Assert.assertTrue(ReflectionHelpers.getField(coreBirthNotificationRegisterActivity, "presenter") instanceof CoreChildRegisterPresenter);
+    }
+
+    @Test
+    public void testGetContext() {
+        Assert.assertEquals(coreBirthNotificationRegisterActivity, coreBirthNotificationRegisterActivity.getContext());
+    }
+
+    @Test
+    public void testHideProgressDialog() {
```

Comment:
Kindly follow the test naming convention `test[MethodName][ExpectedResult][Parameters]`